jest.config.ts

```diff
@@ -276,8 +278,9 @@ const config: Config.InitialOptions = {
       init: {
         // jest project under Sentry organization (dev productivity team)
         dsn: 'https://3fe1dce93e3a4267979ebad67f3de327@sentry.io/4857230',
-        environment: CI ? 'ci' : 'local',
-        tracesSampleRate: 1,
+        // Use production env to reduce sampling of commits on master
+        environment: CI ? (IS_MASTER_BRANCH ? 'production' : 'ci') : 'local',
```

Comment:
Nit: Could we make this `? 'ci:master' : 'ci:pull_request'` or something along those lines?

`production` just feels a little out of place with what it's trying to portray.